Laba1/Laba1.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <iomanip>
+using namespace std;
+int main() {
+	int k;
+	double x;
+	cout << "x = ";
+	cin >> x;
+	cout << "k = ";
+	cin	>> k;
+	int a = 1;
+   double  slag = 1, rez = 1;
+	 while (abs(slag)>= pow(10, -k) ) { 

```

Comment:
'abs' is for integer numbers only. Not sure if the whole algorithm works correct.